content/contributing/development/ui/scss_optimization.rst

```diff
@@ -0,0 +1,203 @@
+=========
+SCSS Tips
+=========
+
+There are many ways to lean and simplify SCSS.
+The first step is to ensablish if custom code is needed at all.
+
+Odoo's webclient has been designed to be modular, meaning that (potentially all) classes can be shared across views.
+Check the code before creating a new class. Chances are that there is already a class or a tag doing exaclty what you're looking for.
+
+
+On top of that, odoo relies on `Bootstrap <https://getbootstrap.com/docs/5.1/getting-started/introduction/>`_ (BS), one of the most complete CSS framework available. The framework has been customised in order to match odoo design (both community and enterprise versions), meaning that you can use any BS class, direclty in odoo and achieve a visual result consistent with our UI.
+
+.. warning::
+   The fact that a class achieve the desired visual result doesn't necessarily mean that's the right one for the job. Be aware of classes triggering JS behaviours, for example.
+
+.. warning::
+   Take care about classes semantic. To apply a **button class** to a **title** is not just sematically wrong, it may also lead to migration issues and visual incostistencies.
```

Comment:
```suggestion
   Take care about classes semantic. To apply a **button class** to a **title** is not just semantically wrong, it may also lead to migration issues and visual inconsistencies.
```